user-guide/Advanced_Functionality/Swarming/EnableSwarming.md

```diff
@@ -47,6 +47,17 @@ To run a prerequisites check using SLNetClientTest tool:
 
    For each prerequisite, the tool will indicate whether the prerequisite is met (*True*) or not (*False*). If you hover over the *Summary* item, you will get a detailed overview of which items cause prerequisites not to be met (e.g. specific scripts, enhanced service connectors, connectors using the obsolete alarm ID format, etc.).
 
+> [!NOTE]
+> Obsolete Engine methods are only included in the prerequisites check from DataMiner 10.5.3 onwards<!--RN 42073-->. If you are using DataMiner 10.5.1 or 10.5.2, these obsolete methods may still be present even if the prerequisite check does not report any issues.
+> The following Engine methods are obsolete and should not be used. Instead, use the corresponding new methods:
+>
+> | Obsolete method | New method |
+> |--|--|
+> | GetAlarmProperty (int, int, string) | GetAlarmProperty (AlarmTreeID, string) |
+> | SetAlarmProperty (int, int, string, string) | SetAlarmProperty (AlarmTreeID, string, string) |
+> | SetAlarmProperties (int, int, string[], string[]) | SetAlarmProperties (AlarmTreeID, string[], string[]) |
+> | AcknowledgeAlarm (int, int, string) | AcknowledgeAlarm (AlarmTreeID, string) |
```

Comment:
@RobbeDGH, I've made some changes based on your feedback. Could you take a look at the parameter types for the new SetAlarmProperty, SetAlarmProperties, and AcknowledgeAlarm methods? I've made an educated guess based on the information you provided for GetAlarmProperty.

Reply:
Thanks for the input, @RobbeDGH! I've corrected the method names and arguments in [commit fcac4aa](https://github.com/SkylineCommunications/dataminer-docs/pull/4265/commits/fcac4aafa6c667cca57277109747c05afb2457e3).